### PR TITLE
fix(web): don't reset context cache with beep 🎼

### PR DESCRIPTION
**Pull request by mcdurdin**
After review with @ermshiperete, we decided to remove this. It is unclear to me why we were ever clearing the context when a beep was triggered by the keyboard. This is legacy from v2.0 of KeymanWeb, and I suspect it's an edge case that no one has ever noticed, but I can't see a reason to keep it.

Test-bot: skip

### CODE REVIEW

File: web/src/app/browser/src/beepHandler.ts [hunk 1]
```diff
@@ -76,11 +75,6 @@ export class BeepHandler {
    * Description  Reset/terminate beep or flash (not currently used: Aug 2011)
    */
   readonly reset = () => {
-    // TODO-web-core: implement for KMX keyboards if needed
-    if (this.keyboardInterface.activeKeyboard instanceof JSKeyboard) {
-      (this.keyboardInterface as JSKeyboardInterface).resetContextCache();
```
Comment (jahorton) [inline web/src/app/browser/src/beepHandler.ts:81]: Note:  this is _not_ a context reset!

This is a reset of the cached-context _construction_ cache.  Whenever a JS keyboard needs to fetch the current context corresponding to a rule, it uses a cache internally to speed up construction across the many conditional checks that occur before a rule finally matches.  It is _this_ cache that is reset - not the user's context, TextStore, or what-have-you.

To verify, note that a project-wide search turns up `KeyboardInterface.cachedContext` and `KeyboardInterface.cachedContextEx`, both of which show up _by far_ the most prominently in the context-related JS-keyboard API methods.

We probably are fine to do this, even still; I can't immediately think of any cases that should invalidate cached portions of the existing context due to a output beep.

**Review by jahorton — APPROVED** [review]

**Review by ermshiperete — APPROVED** [review]
